components/ILIAS/Init/classes/class.ilStartUpGUI.php

```diff
@@ -1304,10 +1305,6 @@ private function showLogout(): void
 
         $tpl->setVariable('LPE', $this->getLogoutPageEditorHTML());
         $tpl->setVariable('TXT_PAGEHEADLINE', $this->lng->txt('logout'));
-        $tpl->setVariable(
-            'TXT_LOGOUT_TEXT',
-            $this->lng->txt('logout_text') . $this->dic['legalDocuments']->logoutText()
```

Comment:
Where do we want to show this text?

`$this->dic['legalDocuments']->logoutText()`